ethereum_history_api/oracles/src/noir/oracles/accountOracle/encode.ts

```diff
@@ -10,7 +10,9 @@ const MAX_ACCOUNT_STATE_LEN = 134;
 export const STATE_PROOF_LEN = PROOF_ONE_LEVEL_LEN * MAX_STATE_PROOF_LEVELS;
 const MAX_STORAGE_PROOF_LEVELS = 7;
 export const STORAGE_PROOF_LEN = PROOF_ONE_LEVEL_LEN * MAX_STORAGE_PROOF_LEVELS;
-export const MAX_RECEIPT_TREE_DEPTH = 7;
+export const MAX_RECEIPT_KEY_NIBBLE_LEN = 6;
+export const MAX_RECEIPT_TREE_DEPTH = MAX_RECEIPT_KEY_NIBBLE_LEN + 1;
+export const MAX_TX_KEY_NIBBLE_LEN = 6;
```

Comment:
The same calculation should be done for `MAX_TRANSACTION_TREE_DEPTH`

Reply:
Done